app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -83,31 +83,21 @@ def start
   def pod_wait
     _log.info("waiting for pod #{pod_full_name} to be available")
 
-    client       = kubernetes_client
-    health_url   = pod_proxy_url(client, INSPECTOR_HEALTH_PATH)
-    http_options = {
-      :use_ssl     => health_url.scheme == 'https',
-      :verify_mode => ext_management_system.verify_ssl_mode,
-      :cert_store  => ext_management_system.ssl_cert_store,
-    }
-
-    # TODO: move this to a more appropriate place (lib)
-    response = pod_health_poll(client, health_url, http_options)
-
-    case response
-    when Net::HTTPOK
+    begin
+      ready = kubernetes_client.get_pod(options[:pod_name], options[:pod_namespace])[:status][:containerStatuses][0][:ready]
```

Comment:
@simon3z As far as I understand, the watch API will hang on the http response waiting for new updates. This means that the worker that is executing the job will hang until the image-inspector scan is complete. It means that parallel scans will be less "parallel" and the scaling will be damaged.
If there is a better way of using the watch API we should consider it but from what I have seen using curl and Kubeclient [1] we have to hang and wait on the connection.

[1]https://github.com/abonas/kubeclient/blob/master/lib/kubeclient/watch_stream.rb#L14